matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/room/create/CreateRoomBodyBuilder.kt

```diff
@@ -112,19 +115,20 @@ internal class CreateRoomBodyBuilder @Inject constructor(
     private suspend fun buildAvatarEvent(params: CreateRoomParams): Event? {
         return params.avatarUri?.let { avatarUri ->
             // First upload the image, ignoring any error
-            tryOrNull {
-                fileUploader.uploadFromUri(
-                        uri = avatarUri,
-                        filename = UUID.randomUUID().toString(),
-                        mimeType = MimeTypes.Jpeg)
+            tryOrNull("Failed to load image") {
```

Comment:
```suggestion
            tryOrNull("Failed to upload image") {
```

Reply:
i added this log for debug, should we keep it ?

Reply:
Yes, let's keep it.
For your information, you can click on "Commit suggestion" directly from GitHub interface if you want to.